server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/csv/IRVHeadersParser.java

```diff
@@ -0,0 +1,128 @@
+/*
+Democracy Developers IRV extensions to colorado-rla.
+
+@copyright 2024 Colorado Department of State
+
+These IRV extensions are designed to connect to a running instance of the raire 
+service (https://github.com/DemocracyDevelopers/raire-service), in order to 
+generate assertions that can be audited using colorado-rla.
+
+The colorado-rla IRV extensions are free software: you can redistribute it and/or modify it under the terms
+of the GNU Affero General Public License as published by the Free Software Foundation, either
+version 3 of the License, or (at your option) any later version.
+
+The colorado-rla IRV extensions are distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along with
+raire-service. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+package au.org.democracydevelopers.corla.csv;
+
+import au.org.democracydevelopers.corla.model.vote.IRVParsingException;
+import au.org.democracydevelopers.corla.model.vote.IRVPreference;
+import org.apache.commons.csv.CSVRecord;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import us.freeandfair.corla.model.Choice;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
```

Comment:
I think it would be useful in the class comment to list what format the CVR header is expected to be in, what is "the expected pattern".

Reply:
Have shifted the example into the class comment and left the precise description at the top of the relevant method.